Sources/Testing/Testing.docc/known-issues.md

```diff
@@ -10,13 +10,135 @@ See https://swift.org/LICENSE.txt for license information
 See https://swift.org/CONTRIBUTORS.txt for Swift project authors
 -->
 
-Highlight known issues when running tests.
```

Comment:
I think this specific phrasing came from our DevPubs friends? So you might want to ask them to look too.